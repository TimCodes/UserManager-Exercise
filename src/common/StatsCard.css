.stats-card {
  height: 200px;
  border: solid lightgray 1px;
}
.stats-card__title {
  font-size: 2.5rem !important;
}

.stats-card__value {
  font-size: 3.5rem !important;
}
