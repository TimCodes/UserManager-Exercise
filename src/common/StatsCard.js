import * as React from "react";
import Card from "@material-ui/core/Card";
import CardContent from "@material-ui/core/CardContent";
import Typography from "@material-ui/core/Typography";
import Divider from "@material-ui/core/Divider";

const StatCard = ({ title, value }) => (
  <Card>
    <CardContent>
      <Typography style={{ fontSize: 25 }}>{title}</Typography>
      <Divider />
      <Typography color="primary" style={{ fontSize: 40 }}>
        {value}
      </Typography>
    </CardContent>
  </Card>
);

export default StatCard;
