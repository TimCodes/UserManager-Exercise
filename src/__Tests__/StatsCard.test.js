import React from 'react';
import renderer from 'react-test-renderer';
import StatsCard from '../common/StatsCard'

test('renders Stats Card', () => {
  const component = renderer.create(
    <StatsCard title="teest" value="test" />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
