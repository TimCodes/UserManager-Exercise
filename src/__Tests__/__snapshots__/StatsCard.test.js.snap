// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Users Header 1`] = `
<div
  className="MuiPaper-root MuiCard-root stats-card MuiPaper-elevation1 MuiPaper-rounded"
>
  <div
    className="MuiCardContent-root"
  >
    <p
      className="MuiTypography-root stats-card__title MuiTypography-body1"
    >
      teest
    </p>
    <hr
      className="MuiDivider-root"
    />
    <p
      className="MuiTypography-root stats-card__value MuiTypography-body1 MuiTypography-colorPrimary"
    >
      test
    </p>
  </div>
</div>
`;
